### DISCUSSION TITLE
Add a default config file location for Windows

### DISCUSSION THREAD

**Discussion by kivikakk** [kivikakk/comrak#432] [Feature requests]
See https://github.com/DanielKeep/cargo-script/blob/614e60e5932e218ebff1e471303eb0d59870d03b/src/platform.rs#L309-L363 for an example of getting the roaming app data folder.

**Comment by coolaj86** [comment]
I'd argue that it's better to have a single location `~/.config/comrak/config` that can be relied on cross-platform rather than many different locations.

Obviously people with Roaming Profiles on an Active Directory domain that switch between various work computers on a whim do exist, but I imagine that once you've narrowed that down to people who enjoy the command line with Unix tools... they're probably used to solving that particular problem in a more Unix-y way.

Likewise, I don't think too many developers on OS X prefer to have their unix tools saving to `~/Library/Application Support` rather than the more familiar `~/.config`.
